tracer/src/Datadog.Trace/Configuration/ImmutableDynamicSettings.cs

```diff
@@ -46,7 +46,7 @@ public bool Equals(ImmutableDynamicSettings? other)
 
             return
                 TraceEnabled == other.TraceEnabled
-             && AppsecStandaloneEnabled == other.AppsecStandaloneEnabled
+             && ApmTracingEnabled == other.ApmTracingEnabled
```

Comment:
What's the difference between `TraceEnabled` and `ApmTracingEnabled`. As far as I understand they are logically, conceptually, and functionally the same, no? 🤔

Reply:
TraceEnabled is a killswitch that disables instrumentation. ApmTracingEnabled is a setting to signal the the APM only traces should not be generated or taken into account.

Reply:
`TraceEnabled` [is no longer a killswitch that disables instrumentation since v3 ](https://github.com/DataDog/dd-trace-dotnet/blob/master/docs/MIGRATING.md#the-client-library-will-still-be-injected-when-dd_trace_enabled0) 🙂